.storybook/blocks/PropertiesTable.jsx

```diff
@@ -0,0 +1,65 @@
+import "@spectrum-css/table";
+
+import { DocsContext, useOf } from "@storybook/blocks";
+import { ResetWrapper } from "@storybook/components";
+import { NAVIGATE_URL } from "@storybook/core-events";
+import { styled } from "@storybook/theming";
+import React, { useContext } from 'react';
+import { Body, Code, LinkableHeading } from "./Typography.jsx";
+
+export const Table = styled.table`
+	padding-block: 10px;
+`;
+
+/**
+ * Displays the modifiable custom properties for a component based on the metadata provided in the story.
+ * The story metadata object is imported from the "metadata.json" file in the component's directory.
+ *
+ * If the metadata object does not contain a "modifiers" array, this component will not render.
+ *
+ * Usage of this doc block within MDX template(s):
+ *  <PropertiesTable />
+ */
+export const PropertiesTable = () => {
+	const context = useContext(DocsContext);
+	const storyMeta = useOf("meta");
+
+	const packageJson = storyMeta?.csfFile?.meta?.parameters?.packageJson ?? {};
+	const metadata = storyMeta?.csfFile?.meta?.parameters?.metadata ?? {};
+
+	if (!packageJson?.name) return;
+	if (!metadata?.modifiers) return;
+
+	return (
+		<ResetWrapper>
+			<LinkableHeading id="modifiable-properties" size="m">
+				<a aria-hidden="true" href="#modifiable-properties" tabIndex="-1" target="_self" onClick={() => {
+					context.channel.emit(NAVIGATE_URL, "#modifiable-properties");
+				}}></a>
+				Modifiable custom properties
+			</LinkableHeading>
+			<Body>
+				These are empty CSS custom property hooks available in this component
+				that enable one-off customizations specific to a product implementation.
+			</Body>
+			<Table className="docblock-properties-table sb-unstyled spectrum-Table spectrum-Table--sizeL spectrum-Table--compact">
```

Comment:
Small suggestion: I'd add `--mod-table-cursor-row-default: auto` to the `.docblock-properties-table` so it doesn't show a pointer.

Reply:
Oh good idea, yes. I added that and pushed up.